spanish-translations.md

```diff
@@ -0,0 +1,19 @@
+# Automatic Spanish Translations
+
+We're happy to announce that from now on we'll deliver Spanish translations to our
+tutorials (script and subtitles). How we made this possible? You may think that
+we hired a professional who takes care of translating the content, well, let me
+proudly say that you're wrong. As programmers, we love to automate processes, so, that's
+exactly what we did. We leverage the powerful translations service of *DeepL*. DeepL is
+a translating cloud service based on an AI algorithm which is surprisingly good. But, 
+this time it wasn't as easy as making an API call, we had to heavily customize our 
+content so it supports our custom MarkDown syntax and keeping the translations as 
+accurate as possible.
```

Comment:
Should we mention that we're using StofDoctrineExtensionsBundle behind the scene? :)

Reply:
I think that goes beyond the scope of this blog post

Reply:
I think it would be cool to share extra technical details in this post. For example, what does it look like when we make the API request to deepl? What settings do we use? How do we manage to translate the text, but not our code blocks? And, once we have a translated script, how do we get back to having translated subtitles? Also, there is our custom "glossary" that we've built. Basically, I think we could extend this post and make it even more interesting.